src/cloud-api-adaptor/podvm-mkosi/README.md

```diff
@@ -52,11 +64,41 @@ make debug # this will rebuild the builder, the binaries and the OS image
 make image-debug # this will only rebuild the OS image
 ```
 
+For Ubuntu debug images:
+```sh
+PODVM_DISTRO=ubuntu make debug
+```
+
 Notice that building a debug image will overwrite any previous existing debug or production image.
 
 For using SSH, create a file `resources/authorized_keys` with your SSH public key. Ensure the permissions
 are set to `0400` for the `authorized_keys` file. SSH access is only possible for the `root` user.
 
+## SFTP image
```

Comment:
This is not related to the ubuntu, could you please extract it into a different commit?